src/website/app/demos/Tooltip/index.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright 2017 CA
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* @flow */
+import { componentTheme } from '../../../../Tooltip/Tooltip';
+import bestPractices from './bestPractices';
+import examples from './examples';
+
+const doc = require('!!react-docgen-loader!../../../../Tooltip/Tooltip');
+
+export default {
+  bestPractices,
+  componentTheme,
+  doc,
+  examples,
+  slug: 'tooltip',
+  title: 'Tooltip',
+  whenHowToUse: `Tooltips are Popovers that display information related to an element when the element receives keyboard focus or the mouse hovers over it. It appears after a small dealy and disappears when \`Escape\` is pressed or on mouse out.
```

Comment:
Add some linebreaks for source legibility, please. Also, "dealy" is one of my favorite typos ever.

Reply:
done